src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,41 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { getFindId } from '../../helper';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const selectedTab = getFindId(tabs, selectedTabId) || tabs[0];
```

Comment:
The logic of substituting the first value in the case of not finding the desired value should be moved to the middle of the helper function.